packages/components/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 -   `TextHighlight`: Convert to TypeScript ([#41698](https://github.com/WordPress/gutenberg/pull/41698)).
 
+### Bug Fix
+
+-   `Popover`: pass missing anchor ref to the `getAnchorRef` callback prop. ([#42076](https://github.com/WordPress/gutenberg/pull/42076))
```

Comment:
Typo 🙂

```suggestion
-   `Popover`: pass missing anchor ref to the `getAnchorRect` callback prop. ([#42076](https://github.com/WordPress/gutenberg/pull/42076))
```

Reply:
Thank you! I've been mispelling ref / rect all day 🤦